stdlib/base.pkl

```diff
@@ -1843,6 +1863,49 @@ class Listing<out Element> extends Object {
   @AlsoKnownAs { names { "unique" } }
   external distinct: Listing<Element>
 
+  /// The first element in this listing.
+  ///
+  /// Throws if this listing is empty.
+  ///
+  /// Facts:
+  /// ```
+  /// new Listing { 1 ; 2 ; 3 }.first == 1
+  /// (import "pkl:test").catch(() -> new Listing {}.first)
+  /// ```
+  external first: Element
+
+  /// Same as [first] but returns [null] if this listing is empty.
```

Comment:
```suggestion
  /// The first element in this listing or [null] if this listing is empty.
  ///
  /// See: [first]
```
nit: not a big fan of `does the same as X` type of documentation. I'd rather just explain it.
Same for `lastOrNull`, `singleOrNull`.

Reply:
I see that, but I also see a point about consistency. This is the common style in the documentation throughout the stdlib (I'm ditto not a fan of "**Tells if** ..." instead of just "Whether ..." for boolean properties, but that's simply the style as is). Punting this suggestion to later work.